pulsar-broker/pom.xml

```diff
@@ -425,6 +425,25 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.gaul</groupId>
+        <artifactId>modernizer-maven-plugin</artifactId>
+        <configuration>
+          <failOnViolations>false</failOnViolations>
```

Comment:
Is it possible we turn this option to `true` and exclude classes we don't handle yet?

I'd prefer we enable the checker and gradually remove the exclusion instead of keeping the check dry-run.

Reply:
it only provides `ignore` packages, i need to list too much packages.
```
          <ignorePackages>
            <ignorePackage>org.apache.pulsar.client</ignorePackage>
          </ignorePackages>
```
i will list packages and start migrate the multi small packages in next PR.